---
title: LOCALDB_ERROR_INTERNAL_ERROR | Microsoft Docs
ms.custom: ''
ms.date: 03/04/2017
ms.prod: sql
ms.reviewer: ''
ms.technology: performance
ms.topic: reference
ms.assetid: 60c7a47b-114c-441b-869e-9d8ebfe4148e
author: stevestein
ms.author: sstein
ms.openlocfilehash: 2393c7da432f2143319588c3c16ea88c3b9f1b65
ms.sourcegitcommit: b87d36c46b39af8b929ad94ec707dee8800950f5
ms.translationtype: MT
ms.contentlocale: es-ES
ms.lasthandoff: 02/08/2020
ms.locfileid: "67995711"
---
# <a name="localdb_error_internal_error"></a>LOCALDB_ERROR_INTERNAL_ERROR
[!INCLUDE[appliesto-ss-xxxx-xxxx-xxx-md](../../includes/appliesto-ss-xxxx-xxxx-xxx-md.md)]
    
## <a name="details"></a>Detalles  
  
|||  
|-|-|  
|Nombre de producto|SQL Server|  
|Id. de evento|264|  
|Origen de eventos|SQL Server Local Database Runtime 12.0|  
|Componente|API de Local Database Runtime|  
|Texto del mensaje|Error inesperado en una llamada al método de API de la instancia de Local Database. Vea el registro de aplicación de Windows para obtener los detalles del error.|  
  
## <a name="explanation"></a>Explicación  
 Se ha producido un error inesperado.  
  
## <a name="user-action"></a>Acción del usuario  
 Vea el registro de eventos para obtener detalles.  
  
  
